File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PlantsUpdateMergeRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PlantsUpdateMergeRequestItem {
    @JsonProperty("MergeDate")
    public String mergeDate;
    @JsonProperty("SourcePlantGroupLabel")
    public String sourcePlantGroupLabel;
    @JsonProperty("TargetPlantGroupLabel")
    public String targetPlantGroupLabel;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PlantsUpdateMergeRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PlantsUpdateMergeRequestItem(
    @JsonProperty("MergeDate")
    val mergeDate: String? = null,
    @JsonProperty("SourcePlantGroupLabel")
    val sourcePlantGroupLabel: String? = null,
    @JsonProperty("TargetPlantGroupLabel")
    val targetPlantGroupLabel: String? = null
)
